src/loader.c

```diff
@@ -577,6 +577,9 @@ static void copy_quoted(char *dst, const char *src, int len)
 	*dst = 0;
 }
 
+/* This function is only called when parsing the auth file, so
+   all users added by this function do not have a dynamic password,
+   by definition. If the password is empty, so be it. */
 static void unquote_add_user(const char *username, const char *password)
```

Comment:
let's change the name of this function to something like `authfile_add_user`

Reply:
done